packages/fx-core/resource/package.nls.json

```diff
@@ -273,6 +273,7 @@
   "plugins.apiConnector.errorMessage.ApiConnectorInputError": "Api connector input empty",
   "plugins.apiConnector.errorMessage.ApiConnectorPathNotExistError": "Path %s not exists. Select a valid folder.",
   "plugins.apiConnector.errorMessage.ApiConnectorCreateFileFail": "Failed to create %s",
+  "plugins.apiConnector.errorMessage.ApiConnectorSampleCodeCreateFailError": "Failed to generate sample code file %s. Reason: %s",
```

Comment:
We may remove the `ApiConnector` prefix and `Error` postfix in `ApiConnectorSampleCodeCreateFailError` since the message is already part of `plugins.apiConnector.errorMessage` name space

Reply:
Thanks.
I will use the key name error.apiConnector.SampleCodeCreateFailError to follow other plugins' error message pattern.

Reply:
We will use another pr to change other error message to align with the pattern.